docs/root/intro/arch_overview/security/rbac_filter.rst

```diff
@@ -102,6 +102,7 @@ The following attributes are exposed to the language runtime:
    connection.uri_san_local_certificate, string, The first URI entry in the SAN field of the local certificate in the downstream TLS connection
    connection.uri_san_peer_certificate, string, The first URI entry in the SAN field of the peer certificate in the downstream TLS connection
    connection.id, uint, Downstream connection ID
+   connection.termination_details, string, The termination details of the connection
```

Comment:
@kyessenov @gargnupur hmm, I'm not sure if we have separate doc for attributes available in CEL but is it really a good idea to add it to the rbac doc? That makes it part of the rbac API and I'm not sure who would ever want to use `termination_details` in access control?

Reply:
We'll need to sort it out later. But let's keep here for now in one table. We need to split them into two parts, one used before request processing and one used after. Termination details could be used for auditing, for example.